docker/release/x86_64/Dockerfile

```diff
@@ -7,7 +7,9 @@ RUN apt-get update;\
 
 #Install iroha
 COPY iroha.deb /tmp/iroha.deb
-RUN apt-get install -y /tmp/iroha.deb; rm -f /tmp/iroha.deb
+RUN dpkg -i /tmp/iroha.deb
+
+RUN apt-get -fy install;rm -f /tmp/iroha.deb
```

Comment:
That approach is more lightweight because `gdebi` adds 50+ Mb to the image size.